SofaKernel/modules/SofaCore/src/sofa/core/objectmodel/BaseData.h

```diff
@@ -245,6 +245,10 @@ class SOFA_CORE_API BaseData : public DDGNode
     bool isSet(const core::ExecParams*) const { return isSet(); }
     bool isSet() const { return m_isSet; }
 
+    /// Return whether the Data has a default value or not
+    bool hasDefaultValue() const { return m_hasDefaultValue; }
+    /// @}
```

Comment:
It seems that the closing bracket is after `getCounter`. Are you sure you want to add this closing curly bracket?